### DISCUSSION TITLE
[Question] How do I redirect post URLs?

### DISCUSSION THREAD

**Discussion by SebastianSolidwork** [GetPublii/Publii#1399] [Help] [answered]
I use the simple theme and want to have posts available under multiple URLs.
How do I do this?

I do not want to but maybe I will reorganize some parts of my website and move some parts. And I want already shared links still be working.

**Comment by dziudek** [comment, marked as answer]
Hi,

It is not an issue, but a topic for discussion - I am converting to discussion

Regarding a problem - please use .htaccess or something like this (depending from your platform) to create 301 redirects for your post from different URLs

**Comment by raramuridesign** [comment]
@SebastianSolidwork If you are using Cloudflare pages, you can use cloudflare redirects, works very well.

**Comment by SebastianSolidwork** [comment]
@dziudek Thanks for the conversion. I was not sure where to place this question as the old forum is read-only now. 
Just know I found out that a redirect would be doable within HTML by suing https://en.wikipedia.org/wiki/Meta_refresh
e.g.: <meta http-equiv="refresh" content="0; url=https://example.com/">
Do you mind adding support for that? I can create an issue for that :-)

For the moment I will use .htacess.

@raramuridesign Thanks, I do not use Cloudflare. I have very simple blog for my sharing my experiences as software tester. Nothing commercially intended.

**Reply by candidexmedia** [reply]
Which host are you using for your site?

**Reply by SebastianSolidwork** [reply]
Netcup.de
And it's https://enteresc.net

**Reply by raramuridesign** [reply]
I am not sure that support will be added for the meta refresh. So probably using htaccess is the way to go.

**Comment by dziudek** [comment]
@SebastianSolidwork - we have no plans to add support for the refresh metatag, because it is much worse solution than redirect via .htaccess - slower and less SEO-friendly

**Reply by SebastianSolidwork** [reply]
Thanks. Then I know what to do.

**Comment by raramuridesign** [comment]
@dziudek Is there a chance you could add support for a redirect file which can be added in the backend of publii

See the docs here for cloudflare.
https://developers.cloudflare.com/pages/configuration/redirects/

I can see this being added under Tools & Plugins, then a section for redirects
Under this choose either cloudflare or htaccess 
Code is generated as required.

This will then solve the redirect issue users might have.
M

**Reply by dziudek** [reply]
@raramuridesign - under Tools > Files Manager you can add any file in root directory, so there is no need to prepare a separate tool, because different platforms can have different requirements for the redirects configuration. With this approach you can edit the file responsible for redirects in normal editor offered by your OS (attached to a specific file extension)

**Reply by raramuridesign** [reply]
@dziudek Thank you. I have just tested this and it works like a charm.
